content/docs/guides/requirements.md

```diff
@@ -14,12 +14,11 @@ aliases = ["/docs/guides/"]
 * [Tensorflow](https://www.tensorflow.org/get_started/)
 * [Ksonnet](https://ksonnet.io/docs/tutorial)
 
-For an end to end example illustrating in details how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
+For an end-to-end example illustrating in detail how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
```

Comment:
The Intel tutorial is out of date at this point. Do we have something up to date we can link to? Or should we just remove it for now?

Reply:
It doesn't really belong in Requirements and #151 will provide a more up-to-date example, so I'll take it out & reach out to the author about updating his post.